crates/forge/src/lib.rs

```diff
@@ -302,5 +291,29 @@ fn check_requirements(output_on_success: bool) -> Result<()> {
             r"universal-sierra-compiler (?<version>[0-9]+.[0-9]+.[0-9]+)",
         ),
     });
-    requirements_checker.check()
+    requirements_checker.check()?;
+
+    let scarb_version = ScarbCommand::version().run()?.scarb;
+    if scarb_version <= MINIMAL_SCARB_VERSION_PREBUILT_PLUGIN {
```

Comment:
Shouldn't that be `<`?
```suggestion
    if scarb_version < MINIMAL_SCARB_VERSION_PREBUILT_PLUGIN {
```

Reply:
Yes absolutely